docs/natspec-format.rst

```diff
@@ -49,7 +49,7 @@ The following example shows a contract and a function using all available tags.
 .. code:: solidity
```

Comment:
I think the statements in the previous paragraph are wrong.

Reply:
A short test confirmed that it is correct. Probably because we use `interfaceFunctions()` to iterate over the things that will eventually be output as natspec (see Natspec.cpp)

Reply:
We do parse tags on internal functions.

Reply:
ah. I guess we check them but we never output anything for it